ui/src/overridden/frontsite/Document/DocumentDetails/DocumentCirculationExtras.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import _get from "lodash/get";
+import _isEmpty from "lodash/isEmpty";
+import { invenioConfig } from "@inveniosoftware/react-invenio-app-ils";
+import { Embed, Header, Button, Divider } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { shelfLink } from "../../../utils";
+
+function canCirculateItem(item) {
+  return invenioConfig.ITEMS.canCirculateStatuses.includes(item.status);
+}
+
+function isItemForReference(item) {
```

Comment:
could you make sure (check with Salome or Lydia) the for reference items should also have map displayed?